generated/docs/DetectorCreationInput.md

```diff
@@ -9,7 +9,7 @@ Name | Type | Description | Notes
 **query** | **str** | A question about the image. | 
 **group_name** | **str** | Which group should this detector be part of? | [optional] 
 **confidence_threshold** | **float** | If the detector&#39;s prediction is below this confidence threshold, send the image query for human review. | [optional]  if omitted the server will use the default value of 0.9
-**config_name** | **str, none_type** | (Advanced usage) If your account has multiple named ML configuration options enabled, you can use this field to specify which one you would like to use. | [optional] 
+**pipeline_config** | **str, none_type** | (Advanced usage) Configuration to instantiate a specific prediction pipeline. | [optional] 
```

Comment:
Should we be manually updating the docs in the `generated/docs` directory?

Reply:
cc: @michael-groundlight who might know more about how these docs are used